nextmv/nextmv/cloud/application.py

```diff
@@ -409,6 +447,53 @@ def list_instances(self) -> list[Instance]:
 
         return [Instance.from_dict(instance) for instance in response.json()]
 
+    def list_managed_inputs(self) -> list[ManagedInput]:
+        """
+        List all managed inputs.
+
+        Returns
+        -------
+        list[ManagedInput]
+            List of managed inputs.
+
+        Raises
+        ------
+        requests.HTTPError
+            If the response status code is not 2xx.
+        """
+
+        response = self.client.request(
+            method="GET",
+            endpoint=f"{self.experiments_endpoint}/inputs",
+        )
+
+        return [ManagedInput.from_dict(managed_input) for managed_input in response.json()]
+
+    def list_scenario_tests(self) -> list[BatchExperimentMetadata]:
+        """
+        List all batch scenario tests. Scenario tests are based on the batch
+        experiments API, so this function returns the same information as
+        `list_batch_experiments`, albeit using a different query parameter.
+
+        Returns
+        -------
+        list[BatchExperimentMetadata]
+            List of scenario tests.
+
+        Raises
+        ------
+        requests.HTTPError
+            If the response status code is not 2xx.
+        """
+
+        response = self.client.request(
+            method="GET",
+            endpoint=f"{self.experiments_endpoint}/batch?",
```

Comment:
Is the trailing `?` necessary here? The underlying `requests` should automatically add it when adding params, right? :thinking: 
Not important though if it works. It just looks a bit odd to me.

Reply:
Leftover from some stuff I was trying. Nice catch, I removed it.